analyzers/fcc/Reader.py

```diff
@@ -251,13 +247,13 @@ def __init__(self):
 
 
         if hasattr(self.cfg_ana, 'pfcharged'):
-            pfcharged  = get_collection(Particle, 'pfcharged', False)
+            pfcharged  = get_collection(Particle, 'pfcharged', sort=False)
```

Comment:
was sort really set to False here at the beginning? this is quite unexpected.

Reply:
I have removed the "sort=". I hope this is what you meant.